fs/vfs/fs_read.c

```diff
@@ -207,6 +211,7 @@ ssize_t file_read(FAR struct file *filep, FAR void *buf, size_t nbytes)
 
   iov.iov_base = buf;
   iov.iov_len = nbytes;
+  uio_init(&uio);
```

Comment:
should we add `iov` and `count` argument to uio_init?

Reply:
done